frontend/src/js/external-forms/form-components/DropzoneBetweenElements.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from "@emotion/styled";
+import { useState } from "react";
+import { DropTargetMonitor } from "react-dnd";
+
+import Dropzone, {
+  PossibleDroppableObject,
+} from "../../ui-components/Dropzone";
+
+interface Props {
+  onDrop: (props: PossibleDroppableObject, monitor: DropTargetMonitor) => void;
+  acceptedDropTypes: string[];
+  top: number;
+  height: number;
+}
+
+const LineHeight = 3;
+
+const Line = styled("div")<{ show: boolean }>`
+  overflow: hidden;
+  display: block;
```

Comment:
What's the need for `display: block` and `overflow: hidden`? I think we should be able to go without those